internal/census/census_job.go

```diff
@@ -61,10 +60,9 @@ func runInternal(ctx context.Context, c *censusJob) error {
 }
 
 // NextRunIn returns the duration until the next job run should be scheduled.
-// We report as ready immediately after a successful run. This doesn't mean that
-// this job will run immediately, only about as often as the configured scheduler interval.
+// Census will run every hour to ensure any interrupted jobs will be re-attempted
 func (c *censusJob) NextRunIn(_ context.Context) (time.Duration, error) {
-	return 0, nil
+	return 3600, nil
```

Comment:
`NextRunIn` returns a `duration` not number of seconds, I believe this will effectively still be `0` since we round to seconds and convert to an int when inserting into the database `int(nextRunIn.Round(time.Second).Seconds()`. 

Rather use `time.Hour` here.